cmd/proxyplane/opv-proxyplane-http.example.json

```diff
@@ -0,0 +1,69 @@
+{
+  "version": 2,
+  "name": "OPV Proxy Plane",
+  "port": 28001,
+  "debug": false,
+  "host": [
+    "127.0.0.1"
+  ],
+  "cache_ttl": 3600,
+  "timeout": "3s",
+  "endpoints": [
+    {
+      "endpoint": "/tokenize",
+      "method": "POST",
+      "backend": [
+        {
+          "host": [
+            "https://httpbin.org"
```

Comment:
is this supposed to be an array?

Reply:
according to `krakend`'s config, this enables sending concurrent requests to multiple upstreams - kind of interesting features